tests/tck/features/expression/LogicalExpression.feature

```diff
@@ -18,4 +20,25 @@ Feature: Logical Expression
       """
     Then the result should be, in any order:
       | count(*) |
-      | 140      |
+      | 12       |
```

Comment:
Is the change in the result expected?

Reply:
Yes, it's expected. The previous result was wrong.